### PR TITLE
Add sbt-header to code style checks

### PR DESCRIPTION
**Pull request by mdedetrich**
Forgot to do this in previous PR

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

File: build.sbt [hunk 1]
```diff
@@ -129,8 +129,8 @@ lazy val `pekko-connectors` = project
     crossScalaVersions := List() // workaround for https://github.com/sbt/sbt/issues/3465
   )
 
-addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll")
-addCommandAlias("checkCodeStyle", ";scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
+addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll; headerCheckAll")
+addCommandAlias("checkCodeStyle", "headerCreateAll; ;scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
```
Comment (He-Pin) [inline build.sbt:133]: headerCreateAll should in `applyCodeStyle`

**Review by He-Pin — CHANGES_REQUESTED** [review]
need to change and add `+`

**Comment by mdedetrich** [comment]
> need to change and add `+`

@He-Pin fixed

**Review by pjfanning — APPROVED** [review]
lgtm

File: build.sbt [hunk 2]
```diff
@@ -129,8 +129,8 @@ lazy val `pekko-connectors` = project
     crossScalaVersions := List() // workaround for https://github.com/sbt/sbt/issues/3465
   )
 
-addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll")
-addCommandAlias("checkCodeStyle", ";scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
+addCommandAlias("applyCodeStyle", ";scalafmtAll; scalafmtSbt; javafmtAll; +headerCheckAll")
```
Comment (He-Pin) [inline build.sbt:132]: +headerCreateAll

File: build.sbt [hunk 3]
```diff
@@ -129,8 +129,8 @@ lazy val `pekko-connectors` = project
...
+addCommandAlias("checkCodeStyle", "+headerCreateAll; ;scalafmtCheckAll; scalafmtSbtCheck; javafmtCheckAll")
```
Comment (He-Pin) [inline build.sbt:133]: +headerCheckAll

**Review by He-Pin — CHANGES_REQUESTED** [review]
I think you need to update it

File: build.sbt (hunk 3 above)
Comment (He-Pin) [inline build.sbt:133]: `verifyCodeStyle` same as pekko

Comment (mdedetrich) [reply build.sbt:133]: I guess we should use a common name, I prefer checkCodeStyle. If you agree should I just make a PR in pekko to change it?

Comment (He-Pin) [reply build.sbt:133]: @mdedetrich you need still update the `+headerCreateAll` to `+headerCheckAll`

Comment (He-Pin) [reply build.sbt:133]: @mdedetrich I'm Ok with both name,but better to keep consistence.

Comment (mdedetrich) [reply build.sbt:133]: Agreed, considering that pekko is the odd one out are you fine with me changing it there?

Comment (mdedetrich) [reply build.sbt:133]: Ah I see, missed it doing it now

Comment (mdedetrich) [reply build.sbt]: Fixed and pushed.

**Review by He-Pin — APPROVED** [review]
lgtm

File: build.sbt (hunk 3 above)
Comment (He-Pin) [reply build.sbt]: @mdedetrich maybe that wil confuse some one contribute to both project?I mean akka & pekko, I saw @dongxuwang did the same too.

**Pull request merged**